static/src/types.ts

```diff
@@ -0,0 +1,39 @@
+export interface ToastOptions {
```

Comment:
Could we somehow move this file into the plugin folder, closer to the other toaster stuff?